### DISCUSSION TITLE
convert_image.py killed

### DISCUSSION THREAD

**Discussion by AlverGant** [NVlabs/instant-ngp#255] [Q&A]
Trying to convert the Tokyo image in full resolution 56718x21450 and the script gets killed.
It works with 6144x2324 resolution

Environment - Pop!_OS 21.10  32GB RAM

**Comment by Tom94** [comment]
Could you be running out of RAM? My computer only manages it due to having a decent amount of swap space available. (A fp32 4-channel 56817x21450 image in a numpy array is already 20 GB. The calculations to convert sRGB->linear need a few additional temporaries, easily pushing toward ~60 GB in total.)

**Reply by AlverGant** [reply]
Yes it is! I saw it using 32GB of RAM and getting killed, I have no swap file configured ...
